source/documents/insert.txt

```diff
@@ -79,6 +79,10 @@ To insert documents into your collection:
                     { "_id" : 8751, "title" : "The Banquet", "author" : "Dante", "copies" : 2 }
                   ]
 
+            .. note::
+
+              If an object ID is not provided by user, Compass will automatically generate an object ID.
```

Comment:
[leaving a few suggestions to help improve tone and be more direct]

Typically, we want to use active voice wherever possible. So, instead of saying 

"If an object ID is not provided by the user..." 

we can say

"If you do not provide an ObjectId..."

Second person is encouraged where it fits.